WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsAndTopPerformersViewController.swift

```diff
@@ -160,6 +160,8 @@ private extension StoreStatsAndTopPerformersViewController {
                 return
             }
 
+            let forceRefresh = forced || vc.lastFullSyncTimestamp == nil
```

Comment:
Nit: I believe that `vc.lastFullSyncTimestamp == nil` is for force refreshing when loading the screen for the first time. I think it will help to add a comment to explain that. 😄

Reply:
Added a comment in https://github.com/woocommerce/woocommerce-ios/pull/7622/commits/42ed04898e2143c263c7763a03174cad125a673d